src/validation.cpp

```diff
@@ -881,9 +882,14 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
         return state.Invalid(TxValidationResult::TX_WITNESS_MUTATED, "bad-witness-nonstandard");
 
     int64_t nSigOpsCost = GetTransactionSigOpCost(tx, m_view, STANDARD_SCRIPT_VERIFY_FLAGS);
-
-    // We only consider policyAsset
-    ws.m_base_fees = fee_map[policyAsset];
+    
+    CAsset feeAsset;
+    if (g_con_any_asset_fees) {
+        feeAsset = fee_map.begin()->first;
```

Comment:
What happens if there's zero, or two or more?

Reply:
I mean, at least explain it in a comment.

Reply:
Added a comment and some TODOs in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/10/commits/65ffeb051fb030ae1b52c124ed4c2843a9bd7571

I don't like how `fee_map` works here, but we'll probably need to change consensus to enforce that only a single asset is used for fee payments per transaction.